test/e2e/config_amd64.go

```diff
@@ -0,0 +1,11 @@
+package integration
+
+var (
+	STORAGE_OPTIONS          = "--storage-driver vfs"
+	ROOTLESS_STORAGE_OPTIONS = "--storage-driver vfs"
+	CACHE_IMAGES             = []string{ALPINE, BB, fedoraMinimal, nginx, redis, registry, infra, labels}
+	nginx                    = "quay.io/baude/alpine_nginx:latest"
+	BB_GLIBC                 = "docker.io/library/busybox:glibc"
+	registry                 = "docker.io/library/registry:2"
+	labels                   = "quay.io/baude/alpine_labels:latest"
```

Comment:
can we change these from baude to a common account?  podmantest perhaps?

Reply:
that's my quay account. i dont see an issue with this

Reply:
YMMV, but I don't think a personal account should be part of testing of a group project.